content/zh/docs/reference/setup-tools/kubeadm/kubeadm-version.md

```diff
@@ -0,0 +1,22 @@
+---
+reviewers:
+- mikedanese
+- luxas
+- jbeda
+title: kubeadm version
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+<!--
+This command prints the version of kubeadm.
+-->
+
+此命令用来查询 kubeadm 的版本。
+
+{{% /capture %}}
+
+{{% capture body %}}
+{{< include "generated/kubeadm_version.md" >}}
```

Comment:
This line requires the "generated/kubeadm_version.md" file, which is not found yet.

Reply:
just now I added "generated/kubeadm_version.md", but this caused conflicts. So I deleted it and  ci passed.